standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -901,7 +901,9 @@ struct AddPartitionsRequest {
   4: required bool ifNotExists,
   5: optional bool needResult=true,
   6: optional string catName,
-  7: optional string validWriteIdList
+  7: optional string validWriteIdList,
+  // use boolean skipFSWrites to decide whether create directory in file system or not
+  8: optional bool skipFSWrites=false
```

Comment:
as far as I know, Hive only allows native table to add partitions, do we need this for partitioned tables?

Reply:
Yes, we do. Because that's one of the requirement of this patch.